client/package.json

```diff
@@ -24,6 +24,7 @@
     "graphql": "14.4.1",
     "i18next": "17.0.4",
     "jss": "10.0.0-alpha.21",
+    "notistack": "^0.8.6",
```

Comment:
Please use lock version instead of version range notation. "0.8.6" instead of "^0.8.6"